usaspending_api/references/serializers.py

```diff
@@ -75,7 +75,6 @@ class Meta:
 
 class RefProgramActivityBriefSerializer(serializers.ModelSerializer):
     prefetchable = False
-    id = serializers.IntegerField(source='ref_program_activity_id')
```

Comment:
It's actually serialized by including the name in `fields` (line 84).  The explicit line defining the serialized field is only needed when the serialized field name differs from the source name.  In fact, DRF throws an error when you define the field unnecessarily that way (I tried).